goosebit/updater/updates.py

```diff
@@ -60,7 +60,7 @@ def dl_endpoint(self):
         return "download_file"
 
     def generate_chunk(
-        self, request: Request, tenant: str, revision: str, dev_id: str
+        self, request: Request, tenant: str,  dev_id: str
```

Comment:
```suggestion
        self, request: Request, tenant: str, dev_id: str
```

Reply:
Good catch, forgot to install pre-commit hooks.